src/android/app/src/main/res/menu/menu_in_game.xml

```diff
@@ -17,6 +17,11 @@
         android:icon="@drawable/ic_controller"
         android:title="@string/emulation_overlay_options" />
 
+    <item
+        android:id="@+id/menu_rotate_screen"
+        android:icon="@drawable/ic_fit_screen"
```

Comment:
A unique icon is a requirement for merging. It doesn't have to be anything particularly elaborate, but it can't just reuse an existing icon